feathr_project/feathr/definition/query_feature_list.py

```diff
@@ -12,19 +12,24 @@ class FeatureQuery(HoconConvertible):
         feature_list: a list of feature names
         key: key of `feature_list`, all features must share the same key
         """
-    def __init__(self, feature_list: List[str], key: Optional[Union[TypedKey, List[TypedKey]]] = None) -> None:
+    def __init__(self, feature_list: List[str], key: Optional[Union[TypedKey, List[TypedKey]]] = None, override_time_delay: Optional[str] = None) -> None:
         self.key = key
         if isinstance(key, TypedKey):
             self.key = [key]
         self.feature_list = feature_list
+        if override_time_delay:
+            self.overrideTimeDelay = override_time_delay
```

Comment:
```suggestion
        self.overrideTimeDelay = override_time_delay
```
small nit: but can be expressed a bit cleaner like this and the suggestion below